nebuly-platform/values.yaml

```diff
@@ -365,9 +365,8 @@ ingestionWorker:
   thresholds:
     subjectClustering: 0.3
     subjectMergeClusters: 0.3
-    intentAssignmentToExistingCluster: 0.87
```

Comment:
This value is still used in the chart, you should also remove the env variable that uses it otherwise the chart will break

Reply:
(assuming that the worker doesn't require that env var anymore)

Reply:
I removed it from the template